app/controllers/api/v1/merchants/search_controller.rb

```diff
@@ -1,17 +1,17 @@
 class Api::V1::Merchants::SearchController < ApplicationController
   def index
```

Comment:
Really cleaned up these controller, I am now calling on a method called filter defined in a module inside of app/model/concerns